app/src/main/res/layout/item_positive_diagnosis.xml

```diff
@@ -64,6 +65,7 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:srcCompat="@drawable/ic_keyboard_arrow_right_gray" />
+        app:srcCompat="@drawable/ic_keyboard_arrow_right_gray"
+        android:contentDescription="@string/item_positive_diagnosis_action_button_content_description" />
```

Comment:
I think I missed it in the first review but the idea is the same as in the comment regarding the "warning icon". There is a text in this view that says "Verified" or "Need verification" so having description for this arrow is redundant.

Reply:
I was unclear on my first pass, but the whole banner is tappable? If so, this can be nulled. If this button is distinct it should have a description.

Reply:
Yes, it's supposed to be clickable but in the v2 it won't be.